.github/CODEOWNERS

```diff
@@ -0,0 +1,22 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+# *       @global-owner1 @global-owner2
+
+# Order is important; the last matching pattern takes the most
+# precedence. When someone opens a pull request that only
+# modifies JS files, only @js-owner and not the global
+# owner(s) will be requested for a review.
+# *.js    @js-owner
+
+# You can also use email addresses if you prefer. They'll be
+# used to look up users just like we do for commit author
+# emails.
+# *.go docs@example.com
+
+# For more examples see https://help.github.com/articles/about-code-owners/
+* @OctopusDeploy/eng-prod
```

Comment:
This one isn't purely an Eng Prod concern as there's an intention to incorporate this kind of functionality into Server. Would you be okay to add @OctopusDeploy/team-core-platform as well?